Makefile

```diff
@@ -30,3 +30,6 @@ check: test
 
 test:
 	go test ./... -v
+
+clean:
+	rm -f bin/fioconfig*
```

Comment:
This is not related to your change. Please, make it a separate PR.
Also, there needs to be a line end at the last line (a matter of your IDE configuration).